multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -626,10 +681,76 @@ func (r *ServiceExportReconciler) updateOrCreateResourceExport(resName string,
 	return nil
 }
 
+func (r *ServiceExportReconciler) getSubsetsFromEndpointSlice(ctx context.Context, req ctrl.Request) ([]corev1.EndpointSubset, error) {
+	epSliceList := &discovery.EndpointSliceList{}
+	err := r.Client.List(ctx, epSliceList, &client.ListOptions{
+		LabelSelector: getEndpointSliceLabelSelector(req.Name),
+		Namespace:     req.Namespace})
+	if err != nil {
+		return nil, err
+	}
+	if len(epSliceList.Items) == 0 {
+		return nil, nil
+	}
+	var subsets []corev1.EndpointSubset
+	for _, eps := range epSliceList.Items {
+		if eps.AddressType == discovery.AddressTypeIPv4 {
+			ports := convertPorts(eps.Ports)
```

Comment:
If we do not really need to export Endpoints, we can save these conversion steps?

Reply:
convertPorts() could be skipped too?